apps/frontend-pwa/src/components/liveSession/QuestionArea.tsx

```diff
@@ -0,0 +1,311 @@
+import { push } from '@socialgouv/matomo-next'
+import { H1 } from '@uzh-bf/design-system'
+import dayjs from 'dayjs'
+import localForage from 'localforage'
+import _without from 'lodash/without'
```

Comment:
move from lodash stuff to only using ramda

Reply:
replaced with ramda function